src/core/scene/a-scene.js

```diff
@@ -296,8 +296,12 @@ var AScene = module.exports = registerElement('a-scene', {
 
         // setTimeout in case the camera is being set dynamically with a setAttribute.
         setTimeout(function checkForCamera () {
-          var sceneCameras = self.querySelectorAll('[camera]');
-          if (sceneCameras.length !== 0) { return; }
+          var camera = self.querySelector('[camera]');
```

Comment:
`isEntity` is probably simplest, instanceof AEntity would work too

querySelector with rotation + position not as reliable because can't guarantee those will be there